infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationLogService.kt

```diff
@@ -773,6 +773,25 @@ constructor(
             .map { it.value.last().number }
             .toSet()
 
+    private fun canSkipLoadingChanges(
```

Comment:
Tekisin itse tuon ehdon toisin päin, eli `loadChanges` eikä `canSkipLoadingChanges` ja vaihtaisin if/else haarat vastaavasti päin vastoin. Lähinnä koska negaatio on ylimäärinen mentaalinen kuorma lukiessa. Tällöin myös tuo tyypin tarkastus menee luonnollisemmaksi (väärä tyyppi -> false).

Periaatteessa myös tuo null->false mappaus on vähän antipattern jonkavoisi korvata käyttöpään null-mappauksella, mutta tässä tapauksessa se kyllä välttää ihan kohtuullisesti duplikointia joten ehkä parempi näin.